addons/resource.language.en_gb/resources/strings.po

```diff
@@ -21292,3 +21292,8 @@ msgstr ""
 msgctxt "#39102"
 msgid "Play Radio"
 msgstr ""
+
+#: xbmc/network/GUIDialogNetworkSetup.cpp
+msgctxt "#39103"
+msgid "Invalid path given. Edit cancelled."
+msgstr ""
```

Comment:
`Unable to configure network location. Invalid path given.`

Reply:
changed.